buildscripts/checkstyle.xml

```diff
@@ -33,6 +33,11 @@
             <property name="eachLine" value="true"/>
         </module>
 
+    <module name="LineLength">
```

Comment:
Does moving this out of TreeWalker mean it runs against generated files which we were avoiding doing?